### PR TITLE
[FEATURE] cb ngsiv2 - update attribute data

### PR DESCRIPTION
**Pull request by iariasleon**
@XavierVal @manucarrace @mcarracedo

### CODE REVIEW

**Comment by jcanonav** [comment]
LGTM

File: iotqatools/cb_v2_utils.py [hunk 1]
```diff
@@ -348,7 +346,10 @@ def __create_attributes(self, entity_context):
 
     def create_entities(self, context, entities_number, attributes_number):
```
Comment (manucarrace) [inline iotqatools/cb_v2_utils.py:347]: Parameter 'context' should not be present. This function is based on the context and context may be different depending on the test.

Comment (iariasleon) [inline iotqatools/cb_v2_utils.py:347]: The context is a dict with the features of the request. See https://github.com/telefonicaid/iot-qa-tools/blob/develop/iotqatools/cb_v2_utils.py#L72

Comment (iariasleon) [inline iotqatools/cb_v2_utils.py:347]: The context comes from step data in behave. See http://pythonhosted.org/behave/tutorial.html#step-data

**Comment by manucarrace** [comment]
LGTM

**Pull request merged**